pages/index.tsx

```diff
@@ -1,17 +1,28 @@
 import Head from "next/head";
-import styles from "../styles/Home.module.css";
-import Button from "../components/loginButton/loginButton";
+import Link from "next/link";
+import Logo from "../components/Logo/logo";
+import styles from "../styles/WelcomePage.module.css";
+
+export default function WelcomePage() {
+  function myFunction() {
```

Comment:
This should be called in a useEffect on first Page render

Reply:
Now it works thanks 🥳